### PR TITLE
[Bugfix-22892] Add detail to "is an integer" 

### PR DESCRIPTION
**Pull request by livecodealice**
Added the detail that `is an integer` will return true if the number and the floor of it are equal, regardless of a decimal point's presence.

### CODE REVIEW

**Comment by livecodepanos** [comment]
@livecode-vulcan review ok 08266da

**Comment by runrevmark** [comment]
@livecode-vulcan review not ok 08266da

**Comment by livecode-vulcan** [comment]
💔 review by @runrevmark not ok 08266dac98b2151c86e16b7775b6a93893c979a0

File: docs/dictionary/operator/is-a.lcdoc [hunk 1]
```diff
@@ -57,7 +57,9 @@ A <value> is a(n):
 * date if it is in one of the formats produced by the <date> or <time>
   functions 
 * <integer(keyword)> if it consists of digits (with an optional leading
```
Comment (runrevmark) [inline docs/dictionary/operator/is-a.lcdoc]: So the definitions for both integer and number are a little misleading. I suggest defining number first, then integer as follows:
```
* number if it is a result of a numeric operation, or a string representing a
  decimal number (e.g. 100.01 or -10), a hexadecimal number (e.g. 0x1FB)
  or a number in scientific notation (e.g. 100.001e-10).
* <integer(keyword)> if it is a number and the number has no (non-zero)
  fractional part (e.g. 100.000 is an integer, but 100.001 is not an integer).
```

File: docs/notes/bugfix-22892.md [hunk 2]
```diff
@@ -0,0 +1 @@
+# Further explained the workings of is an integer
```
Comment (runrevmark) [inline docs/notes/bugfix-22892.md]: I suggest
```
# Corrected the definition of is a number and integer
```

**Review by runrevmark — COMMENTED** [review]
Hopefully my suggestions should make things a little clearer - if you can see a way to improve further do say!

**Review by runrevmark — APPROVED** [review]

**Comment by runrevmark** [comment]
@livecode-vulcan review ok 482277f

**Comment by livecode-vulcan** [comment]
💙 review by @runrevmark ok 482277fc0759c88f9a1ea38d434669576a0576dc

**Comment by livecode-vulcan** [comment]
😎 test success 482277fc0759c88f9a1ea38d434669576a0576dc

* try-community-armv6-android-sdk26_ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-sdk26_ndk16r15/builds/865)
* try-community-armv7-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-armv7-android-ndk16r15/builds/274)
* try-community-arm64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-arm64-android-ndk16r15/builds/274)
* try-community-x86-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86-android-ndk16r15/builds/274)
* try-community-x86_64-android-ndk16r15: [success](https://vulcan.livecode.com/builders/try-community-x86_64-android-ndk16r15/builds/274)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/2391)
* try-community-universal-ios-iphoneos13.7: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.7/builds/0)
* try-community-universal-ios-iphoneos13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos13.5/builds/43)
* try-community-universal-ios-iphonesimulator13.7: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.7/builds/0)
* try-community-universal-ios-iphonesimulator13.5: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator13.5/builds/42)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/2420)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/2407)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/2406)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/2441)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/2381)

**Pull request merged**